### ISSUE TITLE
[Topic] Use of color

### ISSUE THREAD

**Issue by rianrietveld** [wpaccessibility/wp-a11y-docs#130]
URL: https://wpaccessibility.org/docs/topics/design/color/use-of-color
Code: https://github.com/wpaccessibility/wp-a11y-docs/blob/main/docs/topics/design/color/use-of-color.md

Please read [How to contribute](https://wpaccessibility.org/docs/contribute/) before you start.

## Tasks

This content should be reviewed and restructured.
